tools/wptrunner/wptrunner/browsers/content_shell.py

```diff
@@ -48,54 +37,83 @@
 
 
 def check_args(**kwargs):
-    pass
+    require_arg(kwargs, "webdriver_binary")
+
+
+def browser_kwargs(logger, test_type, run_info_data, config, **kwargs):
+    return {"binary": kwargs["binary"],
+            "webdriver_binary": kwargs["webdriver_binary"],
+            "webdriver_args": kwargs.get("webdriver_args"),
+            "debug_info": kwargs["debug_info"]}
 
 
-def browser_kwargs(logger, test_type, run_info_data, config, subsuite, **kwargs):
-    args = []
-    args.append("--ignore-certificate-errors-spki-list=%s" %
+def executor_kwargs(logger, test_type, test_environment, run_info_data, subsuite,
+                    **kwargs):
+    sanitizer_enabled = kwargs.get("sanitizer_enabled")
+    if sanitizer_enabled:
+        test_type = "crashtest"
+    executor_kwargs = base_executor_kwargs(test_type, test_environment, run_info_data,
+                                           subsuite, **kwargs)
+    executor_kwargs["sanitizer_enabled"] = sanitizer_enabled
+    executor_kwargs["close_after_done"] = True
+    executor_kwargs["reuse_window"] = kwargs.get("reuse_window", False)
+
+    capabilities = {
+        "goog:chromeOptions": {
+            "prefs": {
+                "profile": {
+                    "default_content_setting_values": {
+                        "popups": 1
+                    }
+                }
+            },
+            "excludeSwitches": ["enable-automation"],
+            "w3c": True,
+        }
+    }
+
+    chrome_options = capabilities["goog:chromeOptions"]
+    if kwargs["binary"] is not None:
+        chrome_options["binary"] = kwargs["binary"]
+
+    chrome_options["args"] = []
+    chrome_options["args"].append("--ignore-certificate-errors-spki-list=%s" %
         ','.join(chrome_spki_certs.IGNORE_CERTIFICATE_ERRORS_SPKI_LIST))
     # For WebTransport tests.
-    args.append("--webtransport-developer-mode")
+    chrome_options["args"].append("--webtransport-developer-mode")
+    chrome_options["args"].extend([
```

Comment:
We shouldn't need `--enable-experimental-web-platform-features` because it's [a subset of `--enable-blink-test-features`](https://source.chromium.org/chromium/chromium/src/+/main:content/public/common/content_switches.cc;l=344-346;drc=12dde4dd44dfeef4d4580c20f58db79b4a92c9b5;bpv=0;bpt=0). [Same with `enable-blink-features=MojoJS*`](https://source.chromium.org/chromium/chromium/src/+/main:third_party/blink/renderer/platform/runtime_enabled_features.json5;l=2395-2405;drc=5de203c7a0b89622ea5f851ec7fa35c03d7c4690;bpv=0;bpt=0).